### PR TITLE
PROD-2312 course completed screen

### PR DESCRIPTION
**Pull request by vas3a**
https://topcoder.atlassian.net/browse/PROD-2312 - Create Course completed screen - congratulations

You can access this screen by going to https://local.topcoder-dev.com/learn/freeCodeCamp/responsive-web-design/completed ,
BUT: I added a condition to redirect you back to the course details if you didn't complete the course certification. So, in order to view it you'll have to comment out the code in `src-ts\tools\learn\course-completed\CourseCompletedPage.tsx`, lines 52-57, like: 
image

### CODE REVIEW

**Review by brooketopcoder — APPROVED** [review]
Excellent work!

File: src-ts/tools/learn/course-completed/CourseCompletedPage.tsx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    const providerParam: string = routeParams.provider ?? ''
+    const certificationParam: string = routeParams.certification ?? ''
+    const coursePath = getCoursePath(providerParam, certificationParam)
+
+    const {
+        course: courseData,
+        ready: courseDataReady,
+    }: CoursesProviderData = useCoursesProvider(providerParam, certificationParam)
+
+    const { certificateProgress: progress, ready: progressReady }: MyCertificationProgressProviderData = useMyCertificationProgress(profile?.userId, routeParams.provider, routeParams.certification)
```
Comment (testflyjets) [inline src-ts/tools/learn/course-completed/CourseCompletedPage.tsx]: This is a pretty long line -- can you break it?

File: src-ts/tools/learn/course-completed/CourseCompletedPage.tsx [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+                        <div className={styles['course-frame']}>
+                            <div className={styles['content-wrap']}>
+                                <h1>Congratulations!</h1>
+                                <hr />
+                                <div className="body-large">
+                                    You have successfully completed all Assessments for:
+                                </div>
+                                <div className={styles['course-title']}>
+                                    <StarsSvg />
+                                    <CourseTitle size='xl' title={courseData.title} credits={courseData.provider} type='webdev' />
```
Comment (testflyjets) [inline src-ts/tools/learn/course-completed/CourseCompletedPage.tsx]: Is `type='webdev'` hardcoding a value here that the API will/should return?

File: src-ts/tools/learn/course-completed/CourseCompletedPage.tsx [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                                </div>
+                                <hr />
+                                <p>
+                                    Now that you have completed the {courseData.title},
+                                    take a look at our other Topcoder Academy courses.
+                                    To view other courses, press the start a new course button below.
```
Comment (testflyjets) [inline src-ts/tools/learn/course-completed/CourseCompletedPage.tsx]: It seems to me that this should read:

> To view other courses, press the "Start a new course" button below

If the button has a label, we should show the text of that label exactly in this copy.

**Review by brooketopcoder — APPROVED** [review]

**Review by testflyjets — APPROVED** [review]

**Comment by vas3a** [comment]
merged into #PROD-1985_fcc-proto

**Pull request closed without merging**